R/bs4_book.R

```diff
@@ -679,8 +679,10 @@ check_packages <- function(pkgs) {
 preview_book <- function(path = ".", output = "bookdown::bs4_book") {
   old <- setwd(path)
   on.exit(setwd(old))
-
-  render_book("index.Rmd",
+  if (is_empty(index <- get_index_file())) {
+    stop(sprintf('`index.Rmd` or `index.rmd` was not found in path = "%s".', path))
```

Comment:
I see this error message "index.Rmd or index.rmd" has appeared multiple times in this PR, and I'd prefer that we just use "index.Rmd` in the message. In other words, we don't emphasize "index.rmd". This is because I think the vast majority of users would probably use "index.Rmd", and it's not worth their brain memory to remember that "index.rmd" is also supported.